tests/starters/basic-starter/tests/get-prioritized-static-path.test.ts

```diff
@@ -0,0 +1,119 @@
+import { expect } from '@jest/globals';
+import { DrupalClient } from 'next-drupal';
+import { getPrioritizedStaticPathsFromContext } from 'next-acms-basic-starter/lib/get-prioritized-static-paths';
+
+const context = { locales: null, defaultLocale: null };
+
+const ENTITY_TYPES = ['node--article', 'node--place'];
+
+afterEach(() => {
+  jest.restoreAllMocks();
+});
+const client = new DrupalClient('http://🐈');
+
+describe('generate paths on build with menu links prioritization', () => {
+  test('menu links are part of the paths from context', async () => {
+    jest
+      .spyOn(DrupalClient.prototype, 'getStaticPathsFromContext')
+      .mockImplementation(() => {
+        return Promise.resolve([
+          { params: { slug: ['article', '1'] } },
+          { params: { slug: ['article', '2'] } },
+          { params: { slug: ['article', '3'] } },
+          { params: { slug: ['place', 'boston', 'south', '1'] } },
+          { params: { slug: ['place', 'london', '2'] } },
+        ]);
+      });
+    jest.spyOn(DrupalClient.prototype, 'getMenu').mockImplementation(() => {
+      return Promise.resolve({
+        items: [
+          { url: '/' },
+          { url: '/articles' },
+          { url: '/place/boston/south/1' },
+          { url: '/place/london/2' },
+        ],
+        tree: [
+          { url: '/' },
+          { url: '/articles' },
+          { url: '/place/boston/south/1' },
+          { url: '/place/london/2' },
+        ],
+      });
+    });
+    expect(
+      await getPrioritizedStaticPathsFromContext(context, ENTITY_TYPES),
+    ).toStrictEqual([
+      { params: { slug: ['place', 'london', '2'] } },
+      { params: { slug: ['place', 'boston', 'south', '1'] } },
+      { params: { slug: ['article', '1'] } },
+      { params: { slug: ['article', '2'] } },
+      { params: { slug: ['article', '3'] } },
+    ]);
+    expect(client.getStaticPathsFromContext).toBeCalledTimes(1);
+    expect(client.getMenu).toBeCalledTimes(1);
+  });
+
+  test('menu links are not part of the paths from context', async () => {
+    jest
+      .spyOn(DrupalClient.prototype, 'getStaticPathsFromContext')
+      .mockImplementation(() => {
+        return Promise.resolve([
```

Comment:
In the cases where we expect equal output later, can we add the array to a variable? It makes reviewing easier since I don't have to manually confirm that the arrays are the same 😅